generated/google_cloud_longrunning/lib/longrunning.dart

```diff
@@ -96,6 +96,8 @@ final class Operations {
       if (request.filter != null) 'filter': request.filter!,
       if (request.pageSize != null) 'pageSize': '${request.pageSize}',
       if (request.pageToken != null) 'pageToken': request.pageToken!,
```

Comment:
[out of scope for this PR] Is this generated or hand-written? Consider a followup refactoring to pattern conditionals to avoid the `!` operators.

```dart
      if (request.filter case final filter?) 'filter': filter,
      if (request.pageSize != null) 'pageSize': '${request.pageSize}',
      if (request.pageToken case final pageToken?) 'pageToken': pageToken,
```